src/deepsparse/pipeline.py

```diff
@@ -312,6 +298,17 @@ def create(
             **kwargs,
         )
 
+    @classmethod
+    def default_model_for(cls, task: str) -> Optional[str]:
```

Comment:
what's the intended usage for this? note with auto install it may have some unintended side effects like installing the nm-transformers fork a bit early. if it's just to enable optional `model_path` in the cli, let's support a `"default"` special string that will let us get through the config schema

Reply:
Yeah just for optional model path, good idea